mod/groups/views/default/resources/groups/add.php

```diff
@@ -2,8 +2,6 @@
 
 elgg_gatekeeper();
 
-elgg_require_js('elgg/groups/edit');
```

Comment:
The bad is that these dependencies become less visible. The status quo has the added advantage that plugins overriding the view can change the dependency right there without a separate unrequire.

I'm starting to think doing our AMD migrations via extension may be a better idea. I'm gonna go back and read the tickets...